release-notes/2.1/2.1.8/2.1.8-known-issues.md

```diff
@@ -0,0 +1,23 @@
+# .NET Core 2.1.8 Known Issues and Resolutions
+
+## ASP.NET Core 2.1.8
+
+### Microsoft.AspNetCore dependency downgraded in this release
+
+Existing applications may throw a `FileLoadException` after the 2.1.8 release is installed. The stack trace below is
+common.
+```
+Unhandled Exception: System.IO.FileLoadException: Could not load file or assembly 'Microsoft.AspNetCore, Version=2.1.7.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
+   at Program.CreateWebHostBuilder(String[] args)
+```
+
+#### Workarounds
+
+If this issue affects your application, update the project in one of the following ways:
+1. Remove version from AspNetCore.App `<PackageReference Include="Microsoft.AspNetCore" />`

```

Comment:
The note says **.App** but the Include package doesn't include that. Am I just reading it wrong or is it a typo?

Reply:
@natemcmaster this was a typo in the list of workarounds I copied. Correct?

Reply:
oops, yes. 
```suggestion
1. Remove version from AspNetCore.App `<PackageReference Include="Microsoft.AspNetCore.App" />`
```